src/game-logic/README.md

```diff
@@ -0,0 +1,3 @@
+# Game Logic
+
+This directory contains general game logic that extends beyond the scope of a single component.
```

Comment:
i really like the idea of this directory! i don't know how much i love the move of event-handlers.js into here though. i think it's up to you ultimately as it is a design decision and there is no right or wrong, but i am not a fan of the single generic file containing many loosely related things. i would prefer to take a look at what all those event handlers are and see if there is another way to organize them. i also don't know everything in that file right now, but "event handlers" in a web app to me screams "DOM events". 

i think the tl;dr> is just.. I would like to think about how to organize this directory long term. 😄

Reply:
That makes sense to me! I just pushed https://github.com/jeremyckahn/farmhand/commit/00a40f440e354b4e402c7d5c1f7f081c0092d22e, which moves the UI event and peer request handlers to `src/handlers/` and renames them to be a little clearer:

```
src/handlers/
├── peer-events.js
├── README.md
├── ui-events.js
└── ui-events.test.js
```

What do you think of this structure?